<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqManageServersDialog</class>
 <widget class="QDialog" name="pqManageServersDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage Servers</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="addServer" >
       <property name="text" >
        <string>Add Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editServer" >
       <property name="text" >
        <string>Edit Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteServer" >
       <property name="text" >
        <string>Delete Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectAll" >
       <property name="text" >
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>77</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Servers:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QListWidget" name="servers" />
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>pqManageServersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>330</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel" >
     <x>358</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
